docs/pages/management/dynamic-resources/access-list.mdx

```diff
@@ -0,0 +1,304 @@
+---
+title: Creating Access Lists with IaC
+description: Use Infrastructure-as-Code tooling to create Teleport AccessLists.
+---
+
+Access Lists allow Teleport users to be granted long term access to resources
+managed within Teleport. With Access Lists, administrators and access list
+owners can regularly audit and control membership to specific roles and
+traits, which then tie easily back into Teleport's existing RBAC system.
+
+In this guide, we'll follow up on [the IaC users and roles guide]()
+by allowing users with the `manager` role to grant the `support-engineer` role
+to users meeting specific criteria.
+
+Please note that Access Lists can be managed via IaC but Access List memberships
+cannot. The goal of Access Lists is to decentralize granting and reviewing
+access. By allowing managers to grant access within specific guidelines and
+automatically enforcing review, users can request common access rights without
+having to go through the centralized team managing the Teleport IaC.
+This reduces the load on the centralized IaC/security team, ensures the access
+reviewer is aware of the context, reduces the request resolution time, and
+ensures access grants are periodically reviewed.
+
+### Prerequisites
+
+To follow this guide, you must follow first [the basic users and roles IaC guide]().
+We will reuse its users and roles for our Access List.
+
+### Step 1/3 - Write manifests
+
+#### Write the privileged role manifest
+
+We will create a new role `support-engineer` that grants access to production
+servers. The `engineer` role from the previous guide was only granting access to
+`dev` and `staging` servers.
+
+
+<Tabs>
+<TabItem label="tctl">
+
+Create the following `privileged-role.yaml` file:
+
+```yaml
+kind: role
+version: v7
+metadata:
+  name: support-engineer
+spec:
+  allow:
+    logins: ['root', 'ubuntu', '{{internal.logins}}']
+    node_labels:
+      'env': ['production']
+```
+
+</TabItem>
+<TabItem label="operator">
+
+Create the following `privileged-role.yaml` file:
+
+```yaml
+apiVersion: resources.teleport.dev/v5
+kind: TeleportRole
+metadata:
+  name: support-engineer
+spec:
+  allow:
+    logins: [ 'root', 'ubuntu', '{{internal.logins}}' ]
+    node_labels:
+      'env': [ 'production' ]
+```
+
+<Admonition type="note">
+The role version is v5 because the Teleport Kubernetes Operator doesn't support
+yet roles v6 and v7.
+</Admonition>
```

Comment:
We're adding versions support pretty soon so maybe just remove this note now so we don't have to remove it in a couple of weeks?